src/appReview/actions.ts

```diff
@@ -0,0 +1,28 @@
+export enum Actions {
```

Comment:
why do you need an actions file when using redux toolkit? actions can be declared in the slice, i see you already declared some here `export const { inAppRatingSupported, setAppRated } = slice.actions`...so can we remove this file?

you can declare actions that don't introduce state change in the slice by just returning the state. https://github.com/valora-inc/wallet/blob/33cbe56c15c39a8a343f45fe3756908ab19cc129/src/consumerIncentives/saga.ts#L253 here is an example of triggering a saga from an action exported from a slice

Reply:
Should be fixed now!